nautobot/core/celery/task.py

```diff
@@ -1,13 +1,25 @@
 from celery.utils.log import get_task_logger
 from celery import Task
+from celery.worker.request import Request
 
 
 logger = get_task_logger(__name__)
 
 
+class NautobotRequest(Request):
+    def on_timeout(self, soft, timeout):
+        from nautobot.core.tasks import handle_time_limit_exceeded
```

Comment:
I can see why this inner import is needed (avoiding a circular import loop between core.celery.task and core.tasks), but would it be better to just define `handle_time_limit_exceeded` here in this file and avoid the import?

Reply:
I am thinking I could probably just import the task here. I was going with already established pattern of having tasks in `nautobot.{{app_label}.tasks`, but yeah technically nothing stopping us from just defining it here either.